hooks/validators/deny_unsafe_apparmor.go

```diff
@@ -17,8 +17,8 @@ func (v DenyUnsafeAppArmor) Validate(ctx context.Context, pod *corev1.Pod) field
 	var errs field.ErrorList
 
 	for k, v := range pod.Annotations {
-		if strings.HasPrefix(k, corev1.AppArmorBetaContainerAnnotationKeyPrefix) &&
-			v != corev1.AppArmorBetaProfileRuntimeDefault {
+		if strings.HasPrefix(k, corev1.DeprecatedAppArmorBetaContainerAnnotationKeyPrefix) &&
+			v != corev1.DeprecatedAppArmorBetaProfileRuntimeDefault {
```

Comment:
AppArmor Fields API was added in Kubernetes 1.30, instead of annotations.
https://github.com/kubernetes/kubernetes/pull/123435

PSA must take care of this field.